src/components/thumb-carousel/thumb-carousel.scss

```diff
@@ -69,7 +74,7 @@ $carousel-background: lighten($developer-blue, 40%);
       height: 20px;
       flex-shrink: 0;
       // Base 64 encoded quote glyph
-      background: url(data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz4NCjwhLS0gR2VuZXJhdG9yOiBBZG9iZSBJbGx1c3RyYXRvciAxOS4xLjAsIFNWRyBFeHBvcnQgUGx1Zy1JbiAuIFNWRyBWZXJzaW9uOiA2LjAwIEJ1aWxkIDApICAtLT4NCjxzdmcgdmVyc2lvbj0iMS4xIiBpZD0iTGF5ZXJfMSIgeG1sbnM9Imh0dHA6Ly93d3cudzMub3JnLzIwMDAvc3ZnIiB4bWxuczp4bGluaz0iaHR0cDovL3d3dy53My5vcmcvMTk5OS94bGluayIgeD0iMHB4IiB5PSIwcHgiDQoJIHZpZXdCb3g9IjAgMCAxMDAgMjYiIHN0eWxlPSJlbmFibGUtYmFja2dyb3VuZDpuZXcgMCAwIDEwMCAyNjsiIHhtbDpzcGFjZT0icHJlc2VydmUiPg0KPHN0eWxlIHR5cGU9InRleHQvY3NzIj4NCgkuc3Qwe2ZpbGw6IzQyN0VBODt9DQoJLnN0MXtmaWxsOm5vbmU7fQ0KPC9zdHlsZT4NCjxnPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0wLDE0LjRjMC0xLjIsMC4zLTIuMiwxLjItNC4yTDYsMGg0LjNMNy41LDEwLjhjMS4yLDAuOSwxLjksMi4yLDEuOSwzLjZjMCwyLjUtMi4yLDQuNS00LjcsNC41DQoJCUMyLjEsMTguOSwwLDE2LjksMCwxNC40eiBNMTIuOCwxNC40YzAtMS4yLDAuMy0yLjIsMS4yLTQuMkwxOC45LDBoNC4zbC0yLjksMTAuOGMxLjIsMC45LDEuOSwyLjIsMS45LDMuNmMwLDIuNS0yLjIsNC41LTQuNyw0LjUNCgkJQzE0LjksMTguOSwxMi44LDE2LjksMTIuOCwxNC40eiIvPg0KPC9nPg0KPHJlY3QgaWQ9Il94M0NfU2xpY2VfeDNFXyIgY2xhc3M9InN0MSIgd2lkdGg9IjI0IiBoZWlnaHQ9IjE5Ii8+DQo8L3N2Zz4NCg==);
+      background: url(data:image/svg+xml;base64,PHN2ZyB2ZXJzaW9uPSIxLjEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgdmlld0JveD0iMCAwIDEwMCAyNiI+DQo8cGF0aCBmaWxsPSIjNDI3RUE4IiBkPSJNMCwxNC40YzAtMS4yLDAuMy0yLjIsMS4yLTQuMkw2LDBoNC4zTDcuNSwxMC44YzEuMiwwLjksMS45LDIuMiwxLjksMy42YzAsMi41LTIuMiw0LjUtNC43LDQuNSBDMi4xLDE4LjksMCwxNi45LDAsMTQuNHogTTEyLjgsMTQuNGMwLTEuMiwwLjMtMi4yLDEuMi00LjJMMTguOSwwaDQuM2wtMi45LDEwLjhjMS4yLDAuOSwxLjksMi4yLDEuOSwzLjZjMCwyLjUtMi4yLDQuNS00LjcsNC41IEMxNC45LDE4LjksMTIuOCwxNi45LDEyLjgsMTQuNHoiLz4NCjwvc3ZnPg==);
```

Comment:
Ah, found out why: FF applies URL argument rules, so it needs escaping. Does http://jsbin.com/wemopoqaxa/edit?html,css,output work for you?